app/views/static_pages/home.html.erb

```diff
@@ -20,4 +20,8 @@
   <%= render 'static_pages/home/curriculum', courses: @courses %>
 
   <%= render 'static_pages/home/success_stories', cache: true %>
+
+  <% if Rails.env.test? && Feature.enabled?(:test_feature, current_user) %>
```

Comment:
Going to leave this in here, or just for testing / review? Couldn't see in review, probably because `test feature` didn't exist

Reply:
Yeah just leaving this here as a smoke test, it should be fine with the test backing it.